### PR TITLE
Author Tests

### PR DESCRIPTION
**Pull request by derekperkins**
~~Not complete yet~~  Done

### CODE REVIEW

**Comment by derekperkins** [comment]
@bep @digitalcraftsman @moorereason

File: hugolib/author_test.go [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+	"reflect"
+	"testing"
+)
+
+var (
+	defaultAuthorsMap = map[string]interface{}{
+		"derek": map[string]interface{}{
+			"givenName":   "Derek",
+			"familyName":  "Perkins",
+			"displayName": "Derek Perkins",
```
Comment (moorereason) [inline hugolib/author_test.go]: Remove this `displayName` to check that the default `DisplayName` construction works.

Comment (derekperkins) [inline hugolib/author_test.go]: Good suggestion. I also switched over to test the `givenName`/`firstName` and `familyName`/`lastName` aliases

**Comment by derekperkins** [comment]
Ok, 100% code coverage for authors. Squashed and ready for merge

**Pull request closed without merging**